src/services/user.service.js

```diff
@@ -51,6 +51,17 @@ class UserService {
 		return User.findOne({ where: property });
 	}
 
+	/**
+   * @param {integer} managerId id of the manager to be finding
+   * @returns {object} data of the manager found
+	*/
+	static async assignUserManager(managerId) {
+		const managerData = await Querie.assignUserManager(db.user, managerId);
```

Comment:
Why are you using Querie.assig..... Please only use User model don't add Que.....

Reply:
Changed